### PR TITLE
Create event routes

### LINKED ISSUE #33: Create Event Routes
Now that we have our Event model, we need Event routes to actually use them. For this task, please make the following routes in `/backend/routes/event.js` : 

- [x] GET /events - Method that returns all events in the database
- [x] POST /event/add - Method that adds an event to MongoDB (data passed in body). Throws an error if the event doesn't match the existing schema. Initialize volunteers to an empty array.

If time permits: 
- [ ] GET /events/:id - Method that returns the event with the provided id. Throws an error if there is no event associated with that id.

### PR DESCRIPTION
**Pull request by adellevo**

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -19,4 +20,36 @@ app.get("/", (req, res) => {
   res.send("Hello world!");
 });
 
+// #1 - get all events
+app.get("/api/events", async (req, res) => {
```
Comment (vwinstea) [inline backend/index.js]: I think its usually better to omit "api" from api routs
```suggestion
app.get("/events", async (req, res) => {
```

File: backend/index.js [hunk 2]
```diff
@@ -19,4 +20,36 @@ app.get("/", (req, res) => {
...
+    const events = await Event.find({});
+    res.send(events);
+  } catch (error) {
+    res.status(500).send(error.message);
+    console.log(`error is ${error.message}`);
+  }
+});
+
+// #2 - add an event
+app.post("/api/recipe", async (req, res) => {
```
Comment (vwinstea) [inline backend/index.js]: Bootcamp code spotted :)
```suggestion
app.post("/event/add", async (req, res) => {
```

File: backend/index.js [hunk 3]
```diff
@@ -19,4 +20,36 @@ app.get("/", (req, res) => {
...
+  try {
+    const { title, Location, start, end, slots, notes, Volunteers } = req.body;
+    let event = new Event({
+      title,
+      Location,
```
Comment (vwinstea) [inline backend/index.js]: I didn't catch Location and Volunteers were uppercase! Could you possible change the model so those are lowercase?

File: backend/index.js [hunk 4]
```diff
@@ -19,4 +20,36 @@ app.get("/", (req, res) => {
...
+      start,
+      end,
+      slots,
+      notes,
+      Volunteers,
```
Comment (vwinstea) [inline backend/index.js]: We can initialize volunteers to an empty array since there won't be any volunteers when we first make an event

**Review by vwinstea — CHANGES_REQUESTED** [review]
Looking good, although there's a couple changes I'd like before merging this in. Also, if you could move the routes out of index.js and into routes/event.js, that would be great! [Here's a guide from bootcamp on how to do it.](https://h4i.notion.site/Breaking-up-index-js-15f849a3917f4ed7b2f24c946a1965b9) Lmk if you have any questions!

File: backend/index.js (hunk 4 above)
Comment (adellevo) [reply backend/index.js]: Should I do that in the model?

**Review by vwinstea — APPROVED** [review]
Everything looks great! Thanks for making all of those changes!

**Pull request merged**